pkg/kubectl/cmd/convert/convert.go

```diff
@@ -263,6 +263,14 @@ func asVersionedObjects(infos []*resource.Info, specifiedOutputVersion schema.Gr
 
 		converted, err := tryConvert(scheme.Scheme, info.Object, targetVersions...)
 		if err != nil {
+			//Dont fail on not registered error converting objects.
```

Comment:
nit: these comments should have a space. i.e

```suggestion
			// Dont fail on not registered error converting objects.
```